packages/Core/text.candy

```diff
@@ -2,7 +2,7 @@ bool = use "..bool"
 [ifElse] = use "..controlFlow"
 [equals] = use "..equality"
 int = use "..int"
-iterable = use "..iterable"
+# iterable = use "..iterable"
```

Comment:
Why that?

Reply:
`iterable` imports `text`, so this causes a cycle. It was only used in the parameter validation of `fromUtf8`, which is currently commented out for performance reasons